src/workflows-app/components/SubmissionDetailsOutputsTable.ts

```diff
@@ -0,0 +1,98 @@
+  const outputTableData: OutputTableData[] = getOutputTableData(configuredOutputDefinition, sort);
+
+  return div(
+    {
+      style: {
+        backgroundColor: 'rgb(235, 236, 238)',
+        display: 'flex',
+        flex: '1 1 auto',
+        flexDirection: 'column',
+        padding: '1rem 3rem',
+      },
+    },
+    [
+      div(
+        {
+          style: {
+            height: tableHeight({
+              actualRows: configuredOutputDefinition.length,
+              maxRows: configuredOutputDefinition.length,
+              heightPerRow: 50,
+            }),
+          },
+        },
+        [
+          h(AutoSizer, [
+            ({ width, height }) =>
+              h(FlexTable, {
+                'aria-label': 'input definition',
+                width,
+                height,
+                // @ts-expect-error
+                sort,
+                rowCount: configuredOutputDefinition.length,
+                noContentMessage: 'Nothing here yet! Your inputs for this submission will be displayed here.',
+                hoverHighlight: true,
+                rowHeight,
+                rowWidth,
+                columns: [
+                  {
+                    size: { basis: 350 },
+                    field: 'taskName',
+                    headerRenderer: () => h(Sortable, { sort, field: 'taskName', onSort: setSort }, ['Task Name']),
+                    cellRenderer: ({ rowIndex }) => {
+                      return h(TextCell, [outputTableData[rowIndex].taskName]);
+                    },
+                  },
+                  {
+                    size: { basis: 600, grow: 0 },
+                    field: 'variable',
+                    headerRenderer: () => h(Sortable, { sort, field: 'variable', onSort: setSort }, ['Variable']),
+                    cellRenderer: ({ rowIndex }) => {
+                      return h(TextCell, [outputTableData[rowIndex].variable]);
+                    },
+                  },
+                  {
+                    size: { basis: 500, grow: 0 },
+                    field: 'outputTypeStr',
+                    headerRenderer: () => h(Sortable, { sort, field: 'outputTypeStr', onSort: setSort }, ['Type']),
+                    cellRenderer: ({ rowIndex }) => {
+                      return h(TextCell, [outputTableData[rowIndex].outputTypeStr]);
+                    },
+                  },
+                  {
+                    size: { basis: 500, grow: 0 },
+                    field: 'attribute',
+                    headerRenderer: () => 'Attribute',
+                    cellRenderer: ({ rowIndex }) => {
+                      if (outputTableData[rowIndex].destination.type === 'record_update') {
+                        const source: RecordUpdateOutputDestination = outputTableData[rowIndex]
+                          .destination as RecordUpdateOutputDestination;
+                        return h(TextCell, [source.record_attribute]);
```

Comment:
Looks like this was copied from the inputs table. It's confusing to store a "destination" in a variable whose name suggests the opposite ("source").

```suggestion
                        const destination: RecordUpdateOutputDestination = outputTableData[rowIndex]
                          .destination as RecordUpdateOutputDestination;
                        return h(TextCell, [destination.record_attribute]);
```